api/types/events/events.proto

```diff
@@ -2568,3 +2569,28 @@ message SessionRecordingAccess {
     UserMetadata UserMetadata = 3
         [ (gogoproto.nullable) = false, (gogoproto.embed) = true, (gogoproto.jsontag) = "" ];
 }
+
+// SSMRun is emitted after an SSM document completes execution.
+message SSMRun {
```

Comment:
Don't forget to update webapps to add support for this new event. Here's example: https://github.com/gravitational/webapps/pull/1100.